src/main/java/org/cryptomator/ui/preferences/VolumePreferencesController.java

```diff
@@ -141,17 +99,9 @@ public boolean isMountFlagsSupported() {
 		return mountFlagsSupported.getValue();
 	}
 
-	public ObservableValue<Boolean> fuseRestartRequiredProperty() {
-		return fuseRestartRequired;
-	}
-
-	public boolean getFuseRestartRequired() {
-		return fuseRestartRequired.getValue();
-	}
-
 	/* Helpers */
 
-	private class MountServiceConverter extends StringConverter<MountService> {
+	public class MountServiceConverter extends StringConverter<MountService> {
```

Comment:
Why did you make it public? If you use this class somewhere else, extract it to its own file and move it to the common/mount package (or ui/common package, if the first one is not possible).